rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/TaskController.java

```diff
@@ -254,9 +255,11 @@ public ResponseEntity<TaskRepresentationModel> selectAndClaimTask(
     filterCustomIntFields.apply(query);
     sortParameter.apply(query);
 
-    Task selectedAndClaimedTask = taskService.selectAndClaim(query);
+    Optional<Task> selectedAndClaimedTask = taskService.selectAndClaim(query);
 
-    return ResponseEntity.ok(taskRepresentationModelAssembler.toModel(selectedAndClaimedTask));
+    return selectedAndClaimedTask
+        .map(task -> ResponseEntity.ok(taskRepresentationModelAssembler.toModel(task)))
+        .orElseGet(() -> ResponseEntity.notFound().build());
```

Comment:
Please document this behavior in the Javadoc, so that our rest documentation get's updated